5333/src/main/cpp/Robot5333.cpp

```diff
@@ -64,28 +64,28 @@ void Robot::RobotPeriodic() {
   if (enableFOC && drivetrain->GetActiveStrategy() != stratFOC)
     enableFOC = false;
   
-  if (toggleFOC.Update(robotmap.joyGroup.GetButton(ControlMap::activateFOC))) {
+  if (toggleFOC.Update(robotmap.joyGroup.GetButtonRise(ControlMap::activateFOC))) {
```

Comment:
toggleFOC is no longer needed and should be removed (including from Robot5333.h), as JoystickGroup::GetButtonRise() replaces its functionality.